src/components/common/FileUploadPreview.vue

```diff
@@ -85,6 +86,13 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
     await this.computeEmitFileValidity(this.fileUpload)
   }
 
+  /** Programmatically opens the file selection dialog. Can be called externally. */
+  public clickFileInput (): void {
+    // find the find input element in THIS component (not globally)
+    const element = this.$el.querySelector('#file-input') as HTMLElement
+    element.click()
+  }
```

Comment:
Note that my original code used `document.getElementById()` but that will match the FIRST file-input on the page, which means we couldn't render several file input components at the same time. The new code fixes that by selecting the id within THIS component. I tested it locally.